src/opentslm/time_series_datasets/mimiciv_ecg/MimicIVECGDataset.py

```
@@ -0,0 +1,490 @@
+    custom prompts to be provided for evaluation.
+    
+    Requires: pip install wfdb
+    """
+
+    def __init__(
+        self,
+        split: Literal["train", "test", "validation"],
+        EOS_TOKEN: str,
+        format_sample_str: bool = False,
+        time_series_format_function=None,
+        max_samples: int = None,
```

Ruff (0.15.7)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)